src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
+    get info() {
```

Comment:
```suggestion
    y: 0,
    
    get info() {
```
додай відступи між методами так легше читати